console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
```

Comment:
Looks like there are two possibly same files:
- console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js
- console/src/components/Services/ApiExplorer/Analyzer/AnalyseButton.js

And I think we should stick to American spelling (Analyze) as it's more common and already used in many places.

Reply:
Oops, i wanted to do the same, seems like `AnalyseButton` didn't get deleted 
finding now